src/test/java/org/springframework/data/tarantool/repository/support/IntegrationTestWithoutTuples.java

```diff
@@ -0,0 +1,145 @@
+package org.springframework.data.tarantool.repository.support;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.InvalidDataAccessResourceUsageException;
+import org.springframework.data.tarantool.BaseIntegrationTest;
+import org.springframework.data.tarantool.entities.Book;
+import org.springframework.data.tarantool.entities.SampleUser;
+import org.springframework.data.tarantool.repository.BookRepositoryWithTuple;
+import org.springframework.data.tarantool.repository.BookRepositoryWithoutTuple;
+import org.springframework.data.tarantool.repository.SampleUserRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+/**
+ * @author Oleg Kuznetsov
+ */
+@Tag("integration")
+public class IntegrationTestWithoutTuples extends BaseIntegrationTest {
+
+    @Autowired
+    private BookRepositoryWithTuple bookRepositoryWithTuple;
+
+    @Autowired
+    private BookRepositoryWithoutTuple bookRepositoryWithoutTuple;
+
+    @Autowired
+    private SampleUserRepository sampleUserRepository;
+
+    @BeforeAll
+    public static void setUp() throws Exception {
+        tarantoolContainer.executeScript("test_setup.lua").get();
+    }
+
+    @AfterAll
+    public static void tearDown() throws Exception {
+        tarantoolContainer.executeScript("test_teardown.lua").get();
+    }
+
+    @Test
+    @DisplayName("Invocation internal function without tuple annotation should return response as object")
```

Comment:
I don't get that display name, is it rrally necessary? I think it's better to name the test methods according to TCs.

Also, some comments n ncode will be helpful (see @vrogach2020 comments):
"When a method without Tuple annotation is invoked"
"Then the returned result is parsed as an object"